Makefile-os

```diff
@@ -50,7 +50,13 @@ initialize_docker: create_env_file
 # which is wrong for local development.
 	docker compose run --rm --user ${UID}:${GID} web make update_deps
 	docker compose up -d
-	docker compose exec --user olympia web make initialize
+	docker compose exec -u 0 web make initialize
```

Comment:
See comment above that in the Makefile: this change would subtly break local environments in some scenarios, because `initialize` runs `update_deps`, deps which are updated with the user `olympia` everywhere else (as they live in the cloned directory locally). You'd possibly end up with `root` owning some files/directories, making it impossible to update with `olympia` later.

Reply:
I know, I just needed a workaround cause I didn't want to spend too much time. Don't plan to merge this.